src/Common/ProfileEvents.cpp

```diff
@@ -953,6 +953,9 @@ The server successfully detected this situation and will download merged part fr
     M(MemoryWorkerRunElapsedMicroseconds, "Total time spent by MemoryWorker for background work", ValueType::Microseconds) \
     \
     M(ParquetFetchWaitTimeMicroseconds, "Time of waiting fetching parquet data", ValueType::Microseconds) \
+    M(WhereFilterPassedRows, "Number of rows that passed the WHERE filter in the query", ValueType::Number) \
+    M(WhereFilterPassedBytes, "Number of bytes that passed the WHERE filter in the query", ValueType::Number) \
+
```

Comment:
I am not sure if using the `WHERE` clause makes sense here. In query execution we have the `FilterTransform`, which might come from:
1. `WHERE` clause
2. `HAVING` clause
2. `JOIN ON` clause

Maybe rename them to `FilterTransformPassedRows` and `FilterTransformPassedBytes

```suggestion
    M(FilterTransformPassedRows, "Number of rows that passed the WHERE filter in the query", ValueType::Number) \
    M(FilterTransformPassedBytes, "Number of bytes that passed the WHERE filter in the query", ValueType::Bytes) \
```

Reply:
Done